connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/Plugins.java

```diff
@@ -207,6 +208,25 @@ public Converter newConverter(String converterClassOrAlias, AbstractConfig confi
         return config != null ? newConfiguredPlugin(config, klass) : newPlugin(klass);
     }
 
+    public HeaderConverter newHeaderConverter(String converterClassOrAlias, AbstractConfig config) {
+        Class<? extends HeaderConverter> klass;
+        try {
+            klass = pluginClass(
+                    delegatingLoader,
+                    converterClassOrAlias,
+                    HeaderConverter.class
+            );
+        } catch (ClassNotFoundException e) {
+            throw new ConnectException(
+                    "Failed to find any class that implements HeaderConverter and which name matches "
+                            + converterClassOrAlias
+                            + ", available connectors are: "
```

Comment:
let's fix it here and on the Exception message above. "available converters are: "

Reply:
ok